jenkins/pipelines/QE/c/Jenkinsfile

```diff
@@ -1,163 +1,56 @@
 pipeline {
     agent none
     parameters {
-        string(name: 'CBL_VERSION', defaultValue: '3.2.3', description: 'Couchbase Lite Version')
-        string(name: 'PLATFORM', defaultValue: 'ios', description: "The platform to test (e.g. ios, android, windows, linux)")
-        string(name: 'SGW_VERSION', defaultValue: '3.2.3', description: "The version of Sync Gateway to download")
-    }
-    options {
-        timeout(time: 60, unit: 'MINUTES')
-        ansiColor('xterm')
-        timestamps()
+        string(name: 'CBL_VERSION', defaultValue: '3.3.0', description: 'Couchbase Lite Version (build number will be fetched if not provided)')
+        string(name: 'SGW_VERSION', defaultValue: '3.3.0', description: "The version of Sync Gateway to download")
+        string(name: 'TS_DATASET_VERSION', defaultValue: '4.0', description: 'The version of the dataset to use on the test servers')
     }
     stages {
         stage('Init') {
             steps {
                 script {
                     if (params.CBL_VERSION == '') { error "CBL_VERSION is required" }
-                    if (params.PLATFORM == '') { error "PLATFORM is required" }
                     if (params.SGW_VERSION == '') { error "SGW_VERSION is required" }
-                    currentBuild.displayName = "c ${params.CBL_VERSION} (#${currentBuild.number})"
-                    currentBuild.description = "Platform: ${params.PLATFORM}"
+                    
+                    currentBuild.displayName = "C Linux CBL:${params.CBL_VERSION} (#${currentBuild.number})"
+                    currentBuild.description = "SGW: ${params.SGW_VERSION}"
                 }
             }
         }
         stage('Prebuild Servers') {
             steps {
-                script {
-                    def platforms = [
-                        'c_ios', 'c_android', 'c_linux_x86_64', 'c_windows'
-                    ]
-                    def parallelBuilds = [:]
-                    for(p in platforms) {
-                        // Groovy stupidness.  Need to set a local variable here
-                        // to avoid late binding (all jobs use the same platform).
-                        def platform = p
-                        parallelBuilds[platform] = {
-                            build job: 'prebuild-test-server',
-                            parameters: [
-                                string(name: 'TS_PLATFORM', value: platform),
-                                string(name: 'CBL_VERSION', value: params.CBL_VERSION),
-                            ],
-                            wait: true,
-                            propagate: true
-                        }
-                    }
-                    parallel parallelBuilds
-                }
+                build job: 'prebuild-test-server',
+                parameters: [
+                    string(name: 'TS_PLATFORM', value: 'c_linux_x86_64'),
```

Comment:
This is going backwards, progress wise.  Getting these platforms back in should be near the top of the list of things to do.

Reply:
Not having coverage was a big fault of the previous setup, so I was surprised to see this have the platforms and then get rid of them.  It can be after 4.0.2 but it should be prioritized highly.